bundles/org.openhab.binding.pegelonline/README.md

```diff
@@ -0,0 +1,118 @@
+# PegelOnline Binding
+
+Binding to observe water level from german rivers. 
+Data is provided by german **Water-Route and Shipping Agency** [WSV](https://www.pegelonline.wsv.de/).
+Goal is to monitor actual water levels from rivers nearby your home. 
+In case of changing water levels the corresponding warning level is lowered or raised.
+
+## Supported Things
+
+| Label               | Description                                                                     | ID      |
+|---------------------|---------------------------------------------------------------------------------|---------|
+| Measurement Station | Station providing water level measurements                                      | station |
+
+
+## Discovery
+
+In case your home location coordinates are set the discovery will recognize all measurement stations within a radius of 50 km.
+Found Things are added in your Inbox.
+
+
+## Thing Configuration
+
+Thing configuration contains 3 sections
+
+* Station selection
+* Warning Levels of selected station
+* Refresh rate
```

Comment:
Minor: These can be changed into links to the subsections.

Reply:
done